templates/settings/add_apps.html

```diff
@@ -0,0 +1,13 @@
+{% extends "settings/base.html" %}
+
+{% block subtitle %}Add an App{% endblock %}
+
+{% block content %}
+    <form action="/settings/apps/add" method="POST" enctype="multipart/form-data">
+        {% csrf_token %}
+        {{ form.as_p }}
```

Comment:
There's an existing form pattern style with an include per field you can use in the codebase - if you look at the other forms you'll be able to copy it.